style.css

```diff
@@ -1,46 +1,168 @@
-:root{
-    background-color: aliceblue;
-}
-#form-container{
-    width:400px;
-    height: 500px;
-    display:flex;
-    border:3px solid black; 
-    border-radius: 15px;
-    margin:0 auto;
-    background-color: blue;
-}
-#name-label{
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-    font-size: medium;
-    font-weight: bold;
-}
-#number-label{
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-    font-size: medium;
-    font-weight: bold;
-}
-#app{
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-    font-size: medium;
-    font-weight: bold;
-}
-#software{
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-    font-size: medium;
-    font-weight: bold;
-}#adds{
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-    font-size: medium;
-    font-weight: bold;
-}#textarea{
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-    font-size: medium;
-    font-weight: bold;
-}
-#submit{
-cursor: pointer;
-padding: 3;
-color:gray;
-box-shadow: 0 6px black;
-}
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+:root {
+  background-color: #3c3a39;
+  width: 100vw;
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid white;
+  border-radius: 5px;
+  height: 100vh;
+  width: 100vw;
+}
+
+.title {
+  display: flex;
+  font-size: 40px;
+  align-items: center;
+  font-family: 'Poppins', sans-serif;
+  color: aquamarine;
+  width: 300px;
+  margin: auto;
+  justify-content: center;
+}
+
+.description {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  margin: auto;
+  color: white;
+  margin-bottom: 20px;
+  justify-content: center;
+}
+
+.form-container {
```

Comment:
You can get a little more creative by adding a background image to the flex container 😄